### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by ZaneMeroff**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Add File(s)

## What does this PR do?
- Build out skeleton for shortEventCard.js component
- Build out skeleton for FullEventCard.js component
- Update json files to reflect React Native Navigator package

## Related Tickets & Documents
#3 
#5 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
n/a

## Added tests?

- [ ] yes
- [x] no, because they aren't needed yet
- [ ] no, because I need help

## Added to documentation?

- [ ] readme
- [x] no documentation needed

## What gif best describes this PR or how it makes you feel?
🧜‍♂️

### CODE REVIEW

File: components/FullEventCard/FullEventCard.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { StyleSheet, View, Button, Text, ScrollView, CheckBox, TouchableOpacity } from 'react-native';
+
+export default function FullEventCard() {
+  return (
+    <View style={styles.container}>
+      <View style={styles.closeButton}><Button title='X'/></View>
+      <Text style={styles.titleText}>Volleyball</Text>
+      <Text style={styles.text}>Location: Wash Park</Text>
+      <Text style={styles.text}>Start Time: 2:00PM</Text>
```
Comment (ryan-novak) [inline components/FullEventCard/FullEventCard.js:10]: I like incorporating the time onto the cards. We could also implement that onto the pins when a user clicks on those on the map

File: components/FullEventCard/FullEventCard.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      <Text style={styles.text}>End Time: 4:00PM</Text>
+      <Text style={styles.text}>Players Attending: 4</Text>
+      <Text style={styles.text}>Max Players: 8</Text>
+      <Text style={styles.text}>Skill Level: friendly game!</Text>
+      <Text style={styles.text}>Equipment:</Text>
+        <Text style={styles.text}>net</Text>
+        <Text style={styles.text}>volleyball</Text>
```
Comment (ghost) [inline components/FullEventCard/FullEventCard.js:17]: I think in the future we could make these dynamic to update if they have the equipment or not

File: components/FullEventCard/FullEventCard.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    fontWeight: 'bold',
+    marginBottom: 20,
+    color: '#FFF'
+  },
+  text: {
+    fontSize: 20,
+    marginVertical: 5,
+    color: '#FFF'
+  }
+});
```
Comment (ghost) [inline components/FullEventCard/FullEventCard.js:48]: Nice job figuring out the different kind of styling syntax that react-native uses

File: components/ShortEventCard/ShortEventCard.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  return (
+    <View>
+      <Image
+        source={require('../../assets/fake_map.png')}
+        style={{width: 325, height: 300, marginBottom: 20, position: 'relative', top: 30}}/>
+      <ScrollView style={styles.scrollView}>
+        <View style={styles.smallCardContainer}>
+          <Text style={styles.titleText}>Volleyball</Text>
+          <Text style={styles.text}>Location: Wash Park</Text>
+          <Text style={styles.text}>Time: 2:00PM</Text>
```
Comment (ghost) [inline components/ShortEventCard/ShortEventCard.js:14]: I like incorporating the time onto the cards. We could also implement that onto the pins when a user clicks on those on the map

**Review by ghost — COMMENTED** [review]
Looks freaking awesome dude! Nice start!

**Pull request merged**